frontend/packages/dev-console/src/components/topology/__tests__/ApplicationGroupResource.spec.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { Link } from 'react-router-dom';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import TopologyApplicationResourceList from '../TopologyApplicationList';
+import ApplicationGroupResource from '../ApplicationGroupResource';
+import { sampleDeployments } from './topology-test-data';
+
+describe(ApplicationGroupResource.displayName, () => {
+  it('should exists', () => {
+    const wrapper = shallow(
+      <ApplicationGroupResource
+        title="Deployments"
+        resourcesData={sampleDeployments.data}
+        group="a"
+      />,
+    );
+    expect(wrapper.exists()).toBe(true);
```

Comment:
This will always be true. Check this out https://github.com/airbnb/enzyme/issues/1864